samples/SocketsSample/EndPoints/MessagesEndPoint.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
```

Comment:
Running my tool to add license headers (https://github.com/anurse/aspnet-fmt) update coming.